src/main/resources/templates/methodNotification.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script>
+		var getUrl = window.location;
+		window.parent.postMessage("3DS.Notification.Received", getUrl.protocol  "//"  getUrl.host);
```

Comment:
Missing `+`?
```suggestion
		window.parent.postMessage("3DS.Notification.Received", getUrl.protocol + "//"  + getUrl.host);
```

Also: [Location.origin](https://developer.mozilla.org/en-US/docs/Web/API/Location/origin) might be better suited here considered [`postMessage` API](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage)

Reply:
`location.origin` instead of `getUrl.protocol + "//" + getUrl.host`?

Reply:
Yess

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/202/commits/24236697a28b209d3e4da0750547d620751699ce